### PR TITLE
Fix Next button navigation on single page

### PR DESCRIPTION
**Pull request by indytechcook**
Fix Next button navigation on single page

cc @tladendo @vietnogi

### CODE REVIEW

File: src/client/app/submit-work/submit-work.jade [hunk 1]
```diff
@@ -14,6 +14,6 @@ header
 
 main
   - each view in views
-    article(ui-view=view, class=view)
+    article(id=view ui-view=view, class=view)
```
Comment (vietnogi) [inline src/client/app/submit-work/submit-work.jade]: missing comma, also why we need id?  currently the "view" string is too general to be a unique id, try adding a prefix to it.

**Comment by vietnogi** [comment]
1 comment, lgtm

File: src/client/app/submit-work/submit-work.jade (hunk 1 above)
Comment (tladendo) [inline src/client/app/submit-work/submit-work.jade]: `$anchorScroll` navigates to hashes.

**Comment by tladendo** [comment]
lgtm

**Comment by indytechcook** [comment]
@vietnogi i feel like there is a better way to do what I'm doing using states.  I'm open to ideas here as this works but seems to break styling

**Comment by indytechcook** [comment]
This request now depends on #105

**Comment by indytechcook** [comment]
@vietnogi and @tladendo this warrants a new review.  It's quite a bit different now.

**Pull request merged**